w3d_translator/utils.py

```diff
@@ -0,0 +1,15 @@
+# Converts "True" and "False" to their boolean values
+def tf_to_bool(string: str) -> bool:
+    if string.lower() == "true":
+        return True
+    elif string.lower() == "false":
+        return False
+    else:
+        print("Error reading attribute", string)
+        return None
```

Comment:
You don't wanna throw an exception here? This kind of else can be dangerous in a dynamically typed language

Reply:
Good catch, I'll raise a `TranslationError`